clinica/iotools/converter_utils.py

```diff
@@ -30,41 +101,79 @@ def sort_session_list(session_list: List[str]) -> List[str]:
     return [f"ses-M{session[0]}" for session in session_idx]
 
 
-def print_statistics(summary_file, num_subjs, ses_aval, mmt):
+def write_statistics(
+    summary_file: PathLike, num_subjs: int, ses_aval: List[str], mmt: MissingModsTracker
+) -> None:
     """Write statistics file.
 
-    Print to a given input file statistics about missing files and modalities
-    in a dataset.  This method takes in input a MissingModsTracker object (mmt)
+    Write to a given input file statistics about missing files and modalities in a dataset.
+    This method takes in input a MissingModsTracker object (mmt)
     that contains the number of missing modalities for each session and the
     number of missing sessions for each subject.
 
-    Args:
-        summary_file: path of the output file where write.
-        num_subjs: number of subjects.
-        ses_aval: list of sessions available.
-        mmt: object MissingModsTracker
+    Parameters
+    ----------
+    summary_file : PathLike
+        Path of the output file where statistics should be written.
+
+    num_subjs : int
+        Number of subjects.
+
+    ses_aval : list of str
```

Comment:
ses_aval -> ses_avail

Let's fix the original typo since we are at it :+1:

Reply:
Thanks for catching this ! 👍